packages/create-astro/src/index.ts

```diff
@@ -8,7 +8,13 @@ import yargs from 'yargs-parser';
 import { FRAMEWORKS, COUNTER_COMPONENTS } from './frameworks.js';
 import { TEMPLATES } from './templates.js';
 import { createConfig } from './config.js';
-const args = yargs(process.argv);
+
+// NOTE: In the latest version of npm, the default behavior of `npm init` was changed
```

Comment:
it would be nice to put the version number instead of `latest` so context is not lost in the future.

Reply:
done!